source/geod24/concurrency.d

```diff
@@ -2261,10 +2261,10 @@ final public class Channel (T) : Selectable
 
     void close () nothrow
     {
-        this.lock.lock_nothrow();
```

Comment:
`close()` should probably still be synchronized with a mutex, because it clears the internal queues.

Reply:
I just moved it inside the `if` ?